www-servers/lighttpd/lighttpd-1.4.64-r1.ebuild

```diff
@@ -0,0 +1,232 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..4} )
+
+inherit autotools flag-o-matic lua-single readme.gentoo-r1 systemd tmpfiles toolchain-funcs
+
+DESCRIPTION="Lightweight high-performance web server"
+HOMEPAGE="https://www.lighttpd.net https://github.com/lighttpd"
+SRC_URI="https://download.lighttpd.net/lighttpd/releases-1.4.x/${P}.tar.xz"
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm ~arm64 ~hppa ~ia64 ~mips ppc ppc64 ~s390 sparc x86"
+IUSE="brotli dbi doc gnutls ipv6 kerberos ldap lua maxminddb mbedtls mmap mysql nss pcre2 php postgres rrdtool sasl selinux ssl sqlite test webdav xattr zlib zstd"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="lua? ( ${LUA_REQUIRED_USE} )
+	mysql? ( dbi )
+	postgres? ( dbi )
+	sqlite? ( dbi )
+	webdav? ( sqlite )
+"
+
+BDEPEND="virtual/pkgconfig"
+
+COMMON_DEPEND="
+	acct-group/lighttpd
+	acct-user/lighttpd
+	virtual/libcrypt:=
+	brotli? ( app-arch/brotli )
+	dbi? (
+		dev-db/libdbi
+		mysql? ( dev-db/libdbi-drivers[mysql] )
+		postgres? ( dev-db/libdbi-drivers[postgres] )
+		sqlite? ( dev-db/libdbi-drivers[sqlite] )
+	)
+	gnutls? ( net-libs/gnutls )
+	kerberos? ( virtual/krb5 )
+	ldap? ( >=net-nds/openldap-2.1.26 )
+	lua? ( ${LUA_DEPS} )
+	maxminddb? ( dev-libs/libmaxminddb )
+	mbedtls? ( net-libs/mbedtls )
+	nss? ( dev-libs/nss )
+	pcre2? ( dev-libs/libpcre2 )
+	php? ( dev-lang/php:*[cgi] )
+	rrdtool? ( net-analyzer/rrdtool )
+	sasl? ( dev-libs/cyrus-sasl )
+	ssl? ( >=dev-libs/openssl-0.9.7:0= )
+	webdav? (
+		dev-libs/libxml2
+		sys-fs/e2fsprogs
+	)
+	xattr? ( kernel_linux? ( sys-apps/attr ) )
+	zlib? ( >=sys-libs/zlib-1.1 )
+	zstd? ( app-arch/zstd )"
+
+DEPEND="${COMMON_DEPEND}
+	doc?  ( dev-python/docutils )
```

Comment:
Just a nit: the doc? and test? belong in BDEPEND.

Reply:
Happy to make necessary changes.  FYI: this came from copying www-servers/lighttpd/lighttpd-1.4.59-r1.ebuild as a starting point.